aten/src/ATen/native/native_functions.yaml

```diff
@@ -8809,6 +8817,12 @@
 - func: avg_pool2d.out(Tensor self, int[2] kernel_size, int[2] stride=[], int[2] padding=0, bool ceil_mode=False, bool count_include_pad=True, int? divisor_override=None, *, Tensor(a!) out) -> Tensor(a!)
   python_module: nn
   structured: True
+  precomputed:
+    elements: int kH, int kW, int dH, int dW, int padH, int padW
+    replace:
+      - kernel_size -> kH, kW
+      - stride -> dH, dW
+      - padding -> padH, padW
```

Comment:
Here's a thought; what if we change this syntax to:

```
precomputed:
      - kernel_size -> int kH, int kW
      - stride -> int dH, int dW
      - padding -> int padH, int padW
```

More compact and there is no information loss

Reply:
After my change to how `Precompute` stores data internally, this makes sense.